.gitignore

```diff
@@ -70,7 +70,7 @@ typings/
 .LSOverride
 
 # Icon must end with two \r
-Icon
+Icon
```

Comment:
What changed here? :thinking: 

I'm guessing something normalized the newlines to be `\n`, which defeats the intent of this entry. Though I'm not totally sure what this entry is for to be honest (I used to know... some OS-specific file I think).

Reply:
Weird, at any rate something shouldn’t have been pushed. Will fix this